### PR TITLE
bump: playwright to 1.1.1

### LINKED ISSUE #144: Bump Playwright to 1.1.1
Upgrade it to the latest version.

Blocked by https://github.com/microsoft/playwright/issues/2540

types diff which would work:

```diff
diff --git a/src/types.d.ts b/src/types.d.ts
index 1208d12..b115bf8 100644
--- a/src/types.d.ts
+++ b/src/types.d.ts
@@ -1,12 +1,11 @@
 import type {
-  BrowserTypeLaunchOptions,
-  BrowserNewContextOptions,
+  LaunchOptions,
+  BrowserContextOptions,
   WebKitBrowser,
   ChromiumBrowser,
   FirefoxBrowser,
   BrowserType as PlaywrightBrowserType,
-  BrowserTypeConnectOptions,
-  DeviceDescriptor,
+  devices,
 } from 'playwright-core'
 import type { JestDevServerOptions } from 'jest-dev-server'
 import { CHROMIUM, FIREFOX, IMPORT_KIND_PLAYWRIGHT, WEBKIT } from './constants'
@@ -30,22 +29,20 @@ export type SelectorType = {
   name: string
 }
 
-type Devices = { [name: string]: DeviceDescriptor }
-
 export interface Playwright {
   instance: GenericBrowser
-  devices: Devices
+  devices: typeof devices
 }
 
 export type PlaywrightRequireType = BrowserType | typeof IMPORT_KIND_PLAYWRIGHT
 
 export interface Config {
-  launchBrowserApp?: BrowserTypeLaunchOptions
-  context?: BrowserNewContextOptions
+  launchBrowserApp?: LaunchOptions
+  context?: BrowserContextOptions
   exitOnPageError: boolean
   browsers: BrowserType[]
   devices?: string[]
   server?: JestDevServerOptions
   selectors?: SelectorType[]
-  connectBrowserApp?: BrowserTypeConnectOptions
+  connectBrowserApp?: Parameters<GenericBrowser['connect']>[0]
 }

```

### PR DESCRIPTION
**Pull request by mxschmitt**
Should be okay for now. Since I added the alias only for the `devDependencies` it should not cause any issues in the final bundle which we publish to NPM. Also I removed the browser specific packages, since we don't need them in my mind.

Closes #144

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,11 +64,9 @@
...
     "jest": "26.0.1",
     "lint-staged": "10.2.10",
-    "playwright": ">=0.12.1",
-    "playwright-chromium": ">=0.12.1",
-    "playwright-core": ">=0.12.1",
-    "playwright-firefox": ">=0.12.1",
-    "playwright-webkit": ">=0.12.1",
+    "playwright": ">=1.1.1",
+    "playwright-chromium": ">=1.1.1",
+    "playwright-core": "npm:playwright-chromium@1.1.1",
```
Comment (mmarkelov) [inline package.json:69]: shouldn't it be `>=1.1.1`?

Comment (mmarkelov) [reply package.json]: I told about `"playwright-core": ">=1.1.1"` :)

Comment (mxschmitt) [reply package.json]: Thats the workaround for https://github.com/microsoft/playwright/issues/2540

Comment (mmarkelov) [reply package.json]: > Thats the workaround for microsoft/playwright#2540

got it. I think we should keep #144opened while it won't be fixed in **playwright**

**Pull request merged**